libmscore/bracket.cpp

```diff
@@ -41,6 +43,23 @@ Bracket::~Bracket()
       {
       }
 
+//---------------------------------------------------------
+//   playTick
+//---------------------------------------------------------
+
+Fraction Bracket::playTick() const
+      {
+      // Brackets always have a tick value of zero, so play from the start of the first measure in the system that the bracket belongs to.
+      const auto sys = system();
```

Comment:
Also I am not sure whether it makes sense to use `auto` that extensively. It certainly makes sense if the type is rather long to name it explicitly and if that type is absolutely clear from the context, but in such simple cases it might be better for code reader just to see this type name written explicitly.

This is certainly a debatable question though, and it doesn't seem to have any recommendations on it in our coding style documents. Maybe we should add something on this there eventually.

Reply:
I see no harm in using `auto` when the type is clear from the context; it's obviously a system element here because it's simply storing what `system()` is returning.

If you want me to change, it I will, but my own opinion is that using explicit type names when the context is clear unnecessarily clutters the code and can make harder to read.